charts/telemetry-server/values.yaml

```diff
@@ -45,23 +45,26 @@ service:
   type: "ClusterIP"
   port: 9999
 
-
 ingress:
   # If set to false, ingress will not be created
   # Defaults to true
   # options: true, false
   enabled: true
   className: "nginx"
-  annotations: {}
-  # kubernetes.io/ingress.class: nginx
-  # kubernetes.io/tls-acme: "true"
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    kubernetes.io/tls-acme: "true"
+
   hosts:
-  - host: telemetry.example.org #Fully qualified hostname to access telemetry server
+  - host: telemetry.op-aws-eu-dmz-1-ingress.op.suse.org #OpenPlatform wildcard for fully qualified hostname
```

Comment:
This value is deployment specific and instead should come from deployment env settings.